docs/developer/serving/services/ingress-class.md

```diff
@@ -0,0 +1,19 @@
+# Configuring Services custom Ingress Class
+By default, when a Knative Service is created an Ingress Class (`ingress.class`) is automatically chosen for it based on the content of the `networking-config` ConfigMap located inside the `knative-serving` namespace. 
+
+The `networking-config` ConfigMap is configured at installation time and by default if no other implementation is provided this is defaulted to `"istio.ingress.networking.knative.dev"`. Note that it is best to only update this value during the setup of Knative, to avoid getting undefined behavior.
+
+Once configuraded in the `networking-config` ConfigMap, the `ingress.class` will be used for all Knative Services which doesn't specify the `ingress.class` annotation.
+
+## Using the `ingress-class` annotation 
```

Comment:
```suggestion
## Using the ingress class annotation 
```
see https://knative.dev/docs/help/contributor/style-guide/style-and-formatting/#do-not-use-code-formatting-inside-headings

Reply:
done